src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
-import { up } from '../../breakpoint/breakpoint';
+import { up } from '../../style-utils/breakpoint';
+import { isNoneTouch } from '../../style-utils/is-touch';
 
 const ScrollContainer = styled.div`
-  overflow-x: auto;
+  overflow: hidden;
+
+  ${isNoneTouch()} {
+    overflow-x: scroll;
+
+    /**
+     * we push to content box over it's boundaries to have some distance to
+     * to scroll bar
+     */
+    padding-bottom: 16px;
+    margin-bottom: -16px;
```

Comment:
I can see the difference. Is it worth though? Or does it look better for me?

![Screen Shot 2021-12-08 at 18 05 49](https://user-images.githubusercontent.com/1701755/145251737-47231313-41ca-4965-b87a-4156cf853a02.png)

![Screen Shot 2021-12-08 at 18 05 46](https://user-images.githubusercontent.com/1701755/145251730-3a92d3b6-3f19-43ce-aba8-c032a57a7314.png)

Reply:
If you have a "static" scroll bar, the browser adds some extra padding by default. But if you are on Mac with a touchpad, it doesn't. The appearing scroll bar will flow over the arrow.

Reply:
I always forget about the trackpad and I have problems reproducing it. I have an internal trackpad + safari + mouse (also tried to disable bluetooth). Any way I trust you here, let's move forward!